memory/Allocator.c

```diff
@@ -319,41 +319,29 @@ static void connect(struct Allocator_pvt* parent,
     child->parent = parent;
 }
 
+static int findAllocator(struct Allocator_pvt* target, struct Allocator_List** cpp)
```

Comment:
misleading name, this function actually *disconnects* the allocator from the linked list.
Thanks for your effort and sorry it took so long to address it.